specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,63 @@
+# Charge Subcategory
+
+Charge Subcategory indicates what kind of usage or adjustment the row represents. Charge Subcategory is a supplemental detail to the Charge Type. It provide additional context to describe the primary category of charge.
+
+This linkage to the parent ChargeType means that for every entry under ChargeType, there is a corresponding Charge Subcategory that further refines the nature of the charge. It's a nested level of detail that allows users to see not just what type of charge was incurred, but also how that charge interacts with their purchasing agreements and discounts.
+
+When Charge Type is "Usage", the Charge Subcategory specifies if the usage is accounted for by pre-purchased commitments or if the usage has not benefited from these discounts.
+
+When Charge Type is "Adjustment", the Charge Subcategory indicates what kind of after-the-fact adjustment the record represents. Adjustment type is commonly used to identify changes like credits and refunds.
+
+ChargeSubcategory MUST abide by the following requirements:
+
+- The ChargeSubcategory MUST be present in the billing data.
+- ChargeSubcategory is of type String and MUST be one of the allowed values.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Usage" and the charge is covered by a commitment.
+  - When a usage charge is covered by a commitment, ChargeSubcategory MUST be "Commitment Used".
+  - When a commitment is not used within the committed period, ChargeSubcategory MUST be "Commitment Not Used" for the unused usage charge.
+- ChargeSubcategory MUST be null when ChargeType is "Usage" and is not covered by a commitment.
+- ChargeSubcategory MUST NOT be null or empty when ChargeType is "Adjustment".
+  - When ChargeSubcategory is "Refund" or "Credit", the charge MUST be negative.
```

Comment:
Seems like this is not something to enforce here? We have different metrics where its possible not all of them are going to be guaranteed to have this dynamic?

Reply:
@udam-f2  not sure what you are inferring here? Do you want to suggest a change?

Reply:
Instead of having this line here, we should add a "MUST be negative" statement to the columns that we're requiring to be negative. For now, let's remove this from here. We can discuss adding it to other columns if we feel it's required post-1.0.
```suggestion
```